media-source-respec.html

```diff
@@ -576,42 +563,30 @@
               the <a>SourceBufferList</a> returned by <a def-id="sourceBuffers"></a>.</li>
             <li>Destroy all resources for <var>sourceBuffer</var>.</li>
           </ol>
-        </dd>
-
-        <dt>void endOfStream(optional EndOfStreamError error)</dt>
-        <dd>
+        <table class="parameters"><tbody><tr><th>Parameter</th><th>Type</th><th>Nullable</th><th>Optional</th><th>Description</th></tr><tr><td class="prmName">sourceBuffer</td><td class="prmType"><code>SourceBuffer</code></td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptFalse"><span role="img" aria-label="False">✘</span></td><td class="prmDesc"></td></tr></tbody></table><div><em>Return type: </em><code>void</code></div></dd><dt><dfn><code>endOfStream</code></dfn></dt><dd>
           <p>Signals the end of the stream.</p>
 
           <ol class="method-algorithm">
             <li>If the <a def-id="readyState"></a> attribute is not in the <a def-id="open"></a> state then throw an <a def-id="invalid-state-error"></a> exception and abort these steps.</li>
             <li>If the <a def-id="updating"></a> attribute equals true on any <a>SourceBuffer</a> in <a def-id="sourceBuffers"></a>, then throw an <a def-id="invalid-state-error"></a> exception and abort these steps.</li>
             <li>Run the <a def-id="end-of-stream-algorithm"></a> with the <var>error</var> parameter set to <var>error</var>.</li>
           </ol>
-        </dd>
-
-        <dt>void setLiveSeekableRange(double start, double end)</dt>
-        <dd>
+        <table class="parameters"><tbody><tr><th>Parameter</th><th>Type</th><th>Nullable</th><th>Optional</th><th>Description</th></tr><tr><td class="prmName">error</td><td class="prmType"><code>EndOfStreamError</code></td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptTrue"><span role="img" aria-label="True">✔</span></td><td class="prmDesc"></td></tr></tbody></table><div><em>Return type: </em><code>void</code></div></dd><dt><dfn><code>setLiveSeekableRange</code></dfn></dt><dd>
```

Comment:
Many more occurrences in the IDL, attribute, and method of missing types here (void, etc...) I'll stop marking up the diff because I suspect/hope multiple can be fixed easily.
FYI - w3c htmldiff didn't help much. I instead side-by-side compared:
http://rawgit.com/w3c/media-source/70b66f9a34b38a689e0cf006f0b8b4b9533ba929/index.html
versus
http://rawgit.com/w3c/media-source/62842510a4a09ef1e742f4e49d16a854c3e5e954/